joomla/joomla-cms
# [4.0] Feature/com associations model view

### Summary of Changes
This is a cleanup of the HtmlView code to use the model and since there were so many variables missing I have added them as well. Some other cleanup done as well.


### Testing Instructions
1. Apply patch
2. Use the Multilingual associations
3. Check that no errors occur

### Expected result
Multilingual associations work as expected


### Documentation Changes Required
None
